nbs/01a_losses.ipynb

```diff
@@ -62,7 +62,15 @@
     "class BaseLoss():\n",
```

Comment:
<p>Not docments related, but it would be very helpful to include requirements of the <code>loss_cls</code>  function.  What must that function take as input?  What must it return? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>